modules/utxo-lib/src/bitgo/wallet/Unspent.ts

```diff
@@ -360,11 +363,12 @@ export function addWalletUnspentToPsbt(
   /**
    * @deprecated
    */
-  network: Network = psbt.network
+  network: Network = psbt.network,
+  customParams?: { sequenceNumber?: number }
```

Comment:
This would force passing the deprecated parameter. Let's just get rid of it

```suggestion
  params: { sequenceNumber?: number },
```

Reply:
SDK: https://github.com/BitGo/BitGoJS/pull/4043
WP: https://github.com/BitGo/bitgo-microservices/pull/37560